common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -105,17 +105,30 @@ $sequence--border-color: #C8C8C8;
         display: block;
         padding: 0;
         position: relative;
+        text-align: center;
 
-        &:hover, &:focus {
-          background-color: $white;
-          background-repeat: no-repeat;
-          background-position: center 14px;
+        .icon {
+          line-height: 42px;
+          font-size: 90%;
+          color: rgb(80, 80, 80);
+        }
+
+        &.inactive {
+
+          .icon {
+            // color: rgb(160, 160, 160); # Lighter gray, but not sure why?
+            color: rgb(80, 80, 80);
```

Comment:
I guess since this is an `xblock` the normal variables weren't available. The values in our `_variables` file(s) use a tint or shade of the normal color, so I took my best guess with what the outputted RGB value is... and something that looks good.